DynamicFeatureNavigation/DSL/app/src/main/java/com/example/android/dfn/dsl/Const.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.dfn.dsl
+
+/**
+ * Collection of constants required across module boundaries.
+ */
+object Const {
+
+    /**
+     * Package names for dynamic feature modules.
+     */
+    object PackageNames {
```

Comment:
these should be computed from the context and module names, since changing the applicationId will require changes to them as well.

but considering that they might not be needed at all (see other comments) maybe you can just delete the whole `PackageNames` object